src/Maestro/MaestroApplication/MaestroApplication.sfproj

```diff
@@ -66,4 +61,9 @@
   <Target Name="PerformanceTest">
     <!-- Required target for arcade SDK -->
   </Target>
+  <Import Project="..\..\..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.6.7\build\Microsoft.VisualStudio.Azure.Fabric.Application.targets" Condition="Exists('..\..\..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.6.7\build\Microsoft.VisualStudio.Azure.Fabric.Application.targets')" />
+  <Target Name="ValidateMSBuildFiles">
```

Comment:
Is VS moving these around? I've noticed it showing up in a few of my PR's too...